packages/token-providers/package.json

```diff
@@ -61,6 +61,7 @@
     "@smithy/util-defaults-mode-node": "^2.0.10",
     "@smithy/util-retry": "^2.0.0",
     "@smithy/util-utf8": "^2.0.0",
+    "@smithy/querystring-builder": "^2.0.6",
```

Comment:
Why is this dependency needed?

Reply:
It's not needed. I removed in the second commit.